doc/source/tutorial/storage.rst

```diff
@@ -654,6 +654,46 @@ See `Groonga's select command document
 <http://groonga.org/docs/reference/commands/select.html>`_ for more
 details.
 
+How to search by regular expression
+-----------------------------------
+
+In storage mode, you can use a Groonga's functionality from Mroonga as described above.
+Thus, you can search records by using a regular expression via Groonga's functionality.
+
+There are some conditions to use regular expression in Mroonga.
+
+* Create an index with ``TokenRegexp`` tokenizer which is used in ``COMMENT``
+* Use ``*SS`` pragma and ``@~`` in ``WHERE MATCH ... AGAINST`` clause
+
+Here is the example of search by regular expression.
+
+.. code-block:: sql
+
+  CREATE TABLE diaries (
+    content text,
+    FULLTEXT INDEX content_index (content) COMMENT 'tokenizer "TokenRegexp", normalizer "NormalizerAuto"'
+  ) ENGINE=Mroonga DEFAULT CHARSET=utf8mb4;
+
+  INSERT INTO diaries VALUES ('Groonga');
+  INSERT INTO diaries VALUES ('Mroonga');
+  INSERT INTO diaries VALUES ('Rroonga');
+
+  SELECT * FROM diaries WHERE MATCH(content) AGAINST ('*SS content @~ ".?ro+nga"' IN BOOLEAN MODE);
```

Comment:
This pattern can't be evaluated by index.

Reply:
I've rewrite example which use index